src/scripts.js

```diff
@@ -116,10 +116,10 @@ function displayRecipeDetailPage(event) {
     foundRecipe = recipeRepository.recipes.find((current) => {
         return current.id === findId(event)
     })
-    if (user.recipesToCook.length > 0) {
+    if (user.recipesToCook.length > 0 && user.recipesToCook.includes(foundRecipe)) {
```

Comment:
Great job Lauren! Love the "logical and" to check if the recipe is in the user.recipesToCook array to display the delete button